src/playarea/EffectWatcher.ttslua

```diff
@@ -0,0 +1,204 @@
+-- FIND ENEMIES: Cards that are enemies only only when face-down
+-- Here we list them by their id in the metadata
+-- These cases are explicitly defined here because they are not in the metadata
+
+FaceDownEnemies             = { -- Some enemies are Agenda or Act cards that are Enemies in the metadata
+    ['04137a'] = true,    -- Maria DeSilva
+    ['04122'] = true,     -- Harlan Earstone
+    ['04130a'] = true,    -- The other guy
+    ['04209'] = true,     -- The Winged serpent
+}
+AgendaFacedownEnemies       = { -- Some enemies are Agenda or Act cards
+    ["03241"] = true,     -- Empire of the Dead
+    ['01121a'] = true,    -- The Masked Hunter
+    ['50026a'] = true,    -- Nagorath   
+    ['11537a'] = true,    -- Seafloor Leviathan
+}
+
+local PlayermatApi    = require("playermat/PlayermatApi")
+local SearchLib             = require("util/SearchLib")
+local zones                 = require("playermat/Zones")
+
+-- General functions
+
+function onLoad(savedData)
+    self.interactable = false
+end
+
+function showEnemyInfo(colorPrint)
+    -- This function shows how many enemies are in play area and threat areas
+    -- Retrieve objects
+    local enemiesInPlayArea = getEnemiesInPlayArea()
+    local matColors = Player.getAvailableColors()
+    local enemiesInThreatArea = {}
+    for _, matColor in ipairs(matColors) do
+        local adding = getEnemiesInThreatArea(matColor)
+        for _, element in ipairs(adding) do table.insert(enemiesInThreatArea, element) end
+    end
+
+    local num = #enemiesInPlayArea
+    if num > 1 then
+        printToAll('There are ' .. tostring(num) .. ' enemies in the map', colorPrint)
+    elseif num == 1 then
+        printToAll('There is ' .. tostring(num) .. ' enemy in the map', colorPrint)
+    end
+    highlightObjects(enemiesInPlayArea, colorPrint, 5)
+    local num = #enemiesInThreatArea
+    if num > 1 then
+        printToAll('There are ' .. tostring(num) .. ' enemies engaged', colorPrint)
+    elseif num == 1 then
+        printToAll('There is ' .. tostring(num) .. ' enemy engaged', colorPrint)
+    end
+    highlightObjects(enemiesInPlayArea, colorPrint, 5)
+end
+
+function showHandResourceInfo()
+    local COLORS = { White = Color.White, Orange = Color.Orange, Green = Color.Green, Red = Color.Red }
+    -- It iterates all players, -- The mod makes shure the number of players is the same as the number of mats
+    local playerColors = Player.getAvailableColors()
```

Comment:
Should use the playermatApi to get a list of the colors.

Reply:
No, that is built-in in TTS.
I was talking about this though:
https://github.com/argonui/SCED/blob/19b025fb65aea3ac65eba61681250a24998db99c/src/playermat/PlayermatApi.ttslua#L270